systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -253,4 +254,34 @@ public void testClusterWithCustomParameters() {
         assertThat(jsonString, valueOfCmEquals("KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "5"));
         assertThat(jsonString, valueOfCmEquals("KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", "5"));
     }
+    @Test
+    @Resources(value = "../examples/templates/cluster-controller", asAdmin = true)
+    @OpenShiftOnly
```

Comment:
This seems to be focusing on the cluster deployment. Wouldn't it be better to instead use ConfigMap instead of a template? That would make the test suitable for both OpenShift and Kubernetes and add more value. 

The rendering of the template should be already verified by `testStrimziPersistent()`.